src/Core/Scheduler/Providers/3dTiles_Provider.js

```diff
@@ -89,6 +89,9 @@ $3dTiles_Provider.prototype.b3dmToMesh = function b3dmToMesh(data, layer) {
                 mesh.material = new BasicMaterial();
             }
         };
+        if (result.extensions.CESIUM_RTC) {
+            result.scene.RTC = result.extensions.CESIUM_RTC.center;
```

Comment:
I'd prefer the provider to directly implement RTC.
That could be done with something like:
```js
if (result.extensions.CESIUM_RTC) {
    group = new THREE.Group();
    group.position.fromArray(result.extensions.CESIUM_RTC.center); 
    group.children = scene.children;
    scene.children = [group];
}
```